SnowballSwiftExample/SnowballSwiftExample/SnowballSwiftKitHomeView.swift

```diff
@@ -49,7 +49,7 @@ struct SnowballSwiftKitHomeView: View {
                             .padding(.horizontal)
                     }
 
-                    SnowballSwitch(isOn: $switchStateIsOn, title: "Switch Text Label", subtitle: "Provide additional context or helpful information.")
+                    Switch(isOn: $switchStateIsOn, title: "Switch Text Label", subtitle: "Provide additional context or helpful information.")
```

Comment:
we really can't just steal primitive names. is this just a normal switch and we dropped our styling?

Reply:
We can create our own switch style.